Packs/Unshorten_Me/Integrations/unshortenMe/unshortenMe.py

```diff
@@ -0,0 +1,174 @@
+demisto.debug('pack name = unshortenMe, packversion = 1.0.0')
+
+import re
+import urllib3
+
```

Comment:
```suggestion
import demistomock as demisto  # noqa: F401
from CommonServerPython import *  # noqa: F401
```
these two lines need to be added to use all builtin functionalities